src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipeline.scala

```diff
@@ -128,14 +128,21 @@ object ExtractionPipeline {
       linkingEntities = fcReplicate
     )
 
-    // don't need to use files apart from storing them, so we don't assign an output here
+    // don't need to use the following entities apart from storing them, so we don't assign an output here
```

Comment:
IMO we should just remove this comment, because we keep having to update it as the workflow evolves